R/descriptive-table.R

```diff
@@ -1,21 +1,44 @@
 #' Produces counts with respective proportions from specified variables in a dataframe.
+#'
 #' Option to add row and column totals
+#' 
 #' @param df A dataframe (e.g. your linelist)
-#' @param counter A name of the variable (in quotation marks) that you would like to have as rows.
-#' @param grouper A name of the variable (in quotation marks) that you would like to have as columns.
-#' @param multiplier What you would like to have your proportions as (default is per 100).
-#' @param digits The number of decimal places you would like in your proportions (default is 1).
-#' @param proptotal A TRUE/FALSE variable specifying whether you would proportions to be of total cases.
-#' The default is FALSE and returns proportions for each column.
+#' @param counter A name of the variable (in quotation marks) that you would
+#'   like to have as rows.
+#' @param grouper A name of the variable (in quotation marks) that you would
+#'   like to have as columns.
+#' @param multiplier What you would like to have your proportions as (default
+#'   is per 100).
+#' @param digits The number of decimal places you would like in your
+#'   proportions (default is 1).
+#' @param proptotal A TRUE/FALSE variable specifying whether you would
+#'   proportions to be of total cases.The default is FALSE and returns
+#'   proportions for each column.
 #' @param coltotals Add column totals on the end
 #' @param rowtotals Add row totals (only sums counts)
+#' @param single_row if `TRUE` and `grouper = NA`, then the output is flattened
+#'   to a single row so that variables can be concatenated into a data frame.
+#'   Defaults to `FALSE`.
+#' @details The `descriptive()` function, provided a counter and no grouper,
+#'   will return A table giving counts and proportions of each variable in the
```

Comment:
This reads as though descriptive returns multiple tables if you specify multiple variables in counter... Dont think that is the case though eh? Or am I having a special moment?
Otherwise multi descriptive looks good - particularly nice that it accepts both se and nse! 
think it would be too confusing to integrate multi fully in to the descriptive function?

Reply:
Updated the wording to make it less confusing